rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -70,14 +72,15 @@ impl FmcAliasLayer {
         report_boot_status(FmcAliasSubjKeyIdGenerationComplete.into());
 
         // Generate the output for next layer
-        let output = DiceOutput {
+        let mut output = DiceOutput {
             subj_key_pair: key_pair,
             subj_sn,
             subj_key_id,
         };
 
         // Generate Local Device ID Certificate
         Self::generate_cert_sig(env, input, &output, fw_proc_info)?;
+        output.zeroize();
```

Comment:
Are these really "temporaries"? Considering that they're being stored in the certificate, which isn't being zeroized, this seems silly to me.

Reply:
Discussed in the Firmware standup; this is as per FIPS requirement (https://github.com/chipsalliance/caliptra-sw/issues/324)